src/commands/test/lib/help-messages.ts

```diff
@@ -7,27 +7,27 @@ export module Messages {
       export const GenerateUITest = "Generates a Xamarin.UITest project";
 
       export const PrepareAppium = "Creates an artifacts directory with Appium tests.";
-      export const PrepareCalabash = `Creates an artifacts directory with Calabash tests.${os.EOL}` + 
```

Comment:
@erychagov - these bits look like they didn't need to change? What are you using to edit the files? I'm wondering if your tooling is touching the whitespace in some unexpected way?

Reply:
@owenniblock 
I use VSCode. He removes all excess white spaces. That's my setting.